DOCSP-33752: Atlas Device Sync support for geospatial queries (#3102)
mongodb/docs-realm: source/sdk/dotnet/model-data/data-types/geospatials.txt

   :depth: 2
   :class: singlecol

.. versionadded:: 11.1.0

Geospatial data, or "geodata", specifies points and geometric objects on the Earth's 
surface. With the geodata types, you can create queries that check whether a given  
point is contained within a shape. For example, you can find all coffee shops within 
15 km of a specified point.

.. versionchanged:: 11.6.1
   Geospatial data supported in Atlas Device Sync

Realm .NET SDK version 11.16.1 and later adds support for geospatial data
in Atlas Device Sync. This allows you to subscribe to geospatial queries

> in a synced realm. For more information, refer to :ref:`<dotnet-flexible-sync>`.

Reviewer: Nit: the Sync subscriptions page doesn't really have more information about subscribing to geospatial queries. Your new info only says "you can subscribe to geospatial queries" and links back to this page. Maybe we should be more explicit about what more information someone will get if they click the link? i.e.:

> For more information about managing Sync subscriptions, refer to...

If you try to subscribe to a geospatial query with an older version of the SDK,

> you will receive a server error and cause a compensating write.

Reviewer: This is truly a nitpick, but we might want to avoid using "cause" here because that could imply the developer has done something "wrong" to "cause" an error. There could possibly be a blame connotation. Curious for your thoughts on this. If you agree, we should change it on all the pages - but we haven't really discussed this as a team before so I'm not sure if we'd want to make this change.

```suggestion
you will receive a server error with a compensating write.
```

Author: Updated for all
